packages/scoutgame/src/points/refreshBuilderLevels.ts

```diff
@@ -26,5 +28,19 @@ export async function refreshBuilderLevels({ season = getCurrentSeasonStart() }:
       }
     });
   }
-  log.info(`Refreshed builder levels for season ${season} with ${levels.length} builders`);
+  log.info(`Refreshed builder levels for season ${season} with ${levels.length} ranked builders`);
+
+  const levelZeroBuilders = await prisma.userSeasonStats.updateMany({
+    where: {
+      season,
+      userId: {
+        notIn: levels.map((level) => level.builderId)
+      }
+    },
+    data: {
+      level: 0
+    }
+  });
+
+  log.info(`Refreshed builder levels for season ${season} with ${levelZeroBuilders.count} unranked builders`);
```

Comment:
we dont need three info logs every time this happens. you should at least remove the log in the middle so our logs arent very noisy